src/main/java/io/cryostat/net/web/http/api/v2/graph/StartRecordingOnTargetMutator.java

```diff
@@ -132,14 +132,18 @@ public HyperlinkedSerializableRecordingDescriptor getAuthenticated(
         return targetConnectionManager.executeConnectedTask(
                 cd,
                 conn -> {
-                    boolean restart = false;
+                    Boolean restart = false;
+                    String replace = "never";
```

Comment:
If you initialize the variables this way, the `replace` variable will always get ignored since in `shouldRestartRecording`, `replace` will never be null and return here:

```java
        if (replace != null) {
            switch (replace) {
                case "always":
                    return true;
                case "stopped":
                    return restart != null && restart.equals(true);
                case "never":
                    return false;
            }
        }
```